irods/session.py

```diff
@@ -203,3 +204,25 @@ def get_connection_refresh_time(self, **kwargs):
                 connection_refresh_time = -1
 
         return connection_refresh_time
+
+    def get_connection_refresh_time(self, **kwargs):
+        connection_refresh_time = -1
+        
+        connection_refresh_time = int(kwargs.get('refresh_time', -1))
+        if connection_refresh_time != -1:
+            return connection_refresh_time
+
+        try:
+            env_file = kwargs['irods_env_file']
+        except KeyError:
+            return connection_refresh_time
+
+        if env_file is not None:
+            env_file_map = self.get_irods_env(env_file)
+            connection_refresh_time = int(env_file_map.get('irods_connection_refresh_time', -1))
+            if connection_refresh_time < 1:
+                # Negative values are not allowed.
+                logger.debug('connection_refresh_time in {} file has value of {}. Only values greater than 1 are allowed.'.format(env_file, connection_refresh_time))
```

Comment:
should be ">= 1"

Reply:
and setting it to zero?

Reply:
This is basically input validation. If a negative value is specified in irods environment file, it prints a message to that effect and sets the connection refresh time to -1 (the value that indicates connections are NOT refreshed).

Reply:
yes, it's input validation.   an input of zero would show this debug message - is that expected?   and if '1' is allowed, then this message should say '>=1', rather than 'greater than 1'.